src/github.com/getlantern/flashlight/autoupdate/autoupdate.go

```diff
@@ -45,13 +45,14 @@ func Configure(cfg *config.Config) {
 }
 
 func enableAutoupdate(cfg *config.Config) {
-	var err error
-
-	httpClient, err = proxied.ChainedNonPersistent(cfg.CloudConfigCA)
+	rt, err := proxied.ChainedNonPersistent(cfg.CloudConfigCA)
 	if err != nil {
 		log.Errorf("Could not create proxied HTTP client, disabling auto-updates: %v", err)
 		return
 	}
+	httpClient = &http.Client{
```

Comment:
@myleshorton Where underlying libraries require an http.Client, I wrap the RoundTripper from proxied in a client.

While doing this, it occurred to me that we're really not taking advantage of the Timeout capability on http.Client.  Something to consider for another day.